src/Mod/Part/Gui/Command.cpp

```diff
@@ -368,26 +368,7 @@ void CmdPartCommon::activated(int iMsg)
     std::vector<Gui::SelectionObject> Sel =
         getSelection().getSelectionEx(nullptr, App::DocumentObject::getClassTypeId(), Gui::ResolveMode::FollowLink);
 
-    //test if selected object is a compound, and if it is, look how many children it has...
-    std::size_t numShapes = 0;
-    if (Sel.size() == 1){
-        numShapes = 1; //to be updated later in code, if
-        Gui::SelectionObject selobj = Sel[0];
-        TopoDS_Shape sh = Part::Feature::getShape(selobj.getObject());
-        while (numShapes==1 && sh.ShapeType() == TopAbs_COMPOUND) {
-            numShapes = 0;
-            TopoDS_Iterator it(sh);
-            TopoDS_Shape last;
-            for (; it.More(); it.Next()) {
-                ++numShapes;
-                last = it.Value();
-            }
-            sh = last;
-        }
-    } else {
-        numShapes = Sel.size();
-    }
-    if (numShapes < 2) {
+    if (Sel.size() < 1) {

```

Comment:
Weird expression!

The minimum value `size()` can return is 0. So, `Sel.size() < 1` is equal to `Sel.size() == 0` and this means the array is empty. So, write: `Sel.empty()`